elide-graphql/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java

```diff
@@ -563,22 +562,38 @@ private ConnectionContainer replaceObjects(Environment context) {
         return sort.map(Sorting::parseSortRule);
     }
 
+    private MultivaluedHashMap<String, String> getQueryParams(Optional<String> typeName, String filterStr) {
+        return new MultivaluedHashMap<String, String>() {
+            {
+                String filterKey = "filter";
+                if (typeName.isPresent()) {
+                    filterKey += "[" + typeName + "]";
+                }
+                put(filterKey, Arrays.asList(filterStr));
+            }
+        };
+    }
+
     private Optional<FilterExpression> buildFilter(String typeName,
                                                    Optional<String> filter,
                                                    RequestScope requestScope) {
         // TODO: Refactor FilterDialect interfaces to accept string or List<String> instead of (or in addition to?)
         // query params.
         return filter.map(filterStr -> {
-            MultivaluedMap<String, String> queryParams = new MultivaluedHashMap<String, String>() {
-                {
-                    put("filter[" + typeName + "]", Arrays.asList(filterStr));
-                }
-            };
+            String errorMessage = "";
```

Comment:
Unnecessary initialization.
```suggestion
            String errorMessage;
```